package.json

```diff
@@ -109,6 +109,7 @@
   "scripts": {
     "prebuild": "shx rm -rf dist",
     "build": "concurrently 'yarn:build:*'",
+    "watch": "concurrently -n watch: 'yarn:build:* --watch'",
```

Comment:
could this be `watch-build` or `build-watch`?

Reply:
Both sound fine to me.

Reply:
Let's make it `build-watch` then:
```suggestion
    "build-watch": "concurrently -n watch: 'yarn:build:* --watch'",
```